.storybook/typings.d.ts

```diff
@@ -3,4 +3,13 @@ declare module '*.md' {
     export = content;
 }
 
-declare function takeScreenshot(id: string): void;
+type TakeScreenshotParams = {
+    action?: string,
+    id: string,
+    selector?: string,
+    userInput?: string,
+};
+
+declare async function takeScreenshot(id: string): Promise<TakeScreenshotParams>;
+
+declare async function takeScreenshotAfterInput(id: string, selector: string, action: string): Promise<TakeScreenshotParams>;
```

Comment:
I think the types for these functions should be:
```
declare async function takeScreenshot(id: string): Promise<Buffer>;
```
and
```
declare async function takeScreenshotAfterInput(id: string, selector?: string, action?: string = 'click', userInput?: string): Promise<Buffer>;
```
We would no longer need `TakeScreenshotParams`.

Reply:
I'll wait for your pr